run_tests

```diff
@@ -27,10 +27,10 @@ for model in $centos_model $ubuntu_model; do
 
 	# wait for deploy to finish
 	sleep 10
-	juju wait-for application slurmctld --query='status=="active"' --timeout=9m
-	juju wait-for application slurmdbd --query='status=="active"' --timeout=9m
-	juju wait-for application slurmrestd --query='status=="active"' --timeout=9m
-	juju wait-for application slurmd --query='status=="active" || status=="blocked"' --timeout=9m
+	juju wait-for unit slurmctld/0 --query='agent-status=="idle"' --timeout=9m
+	juju wait-for unit slurmdbd/0 --query='agent-status=="active"' --timeout=9m
+	juju wait-for unit slurmrestd/0 --query='agent-status=="active"' --timeout=9m
+	juju wait-for unit slurmd/0 --query='agent-status=="active" || agent-status=="blocked"' --timeout=9m
```

Comment:
Possibly that the `workload-status` is what has a state of `active` and not the `agent-status`?

Reply:
I think I missed something on my commit... I am still a bit confused with this `wait-for` command. But what I could deduce is that `agent-status` is the status of the agent and `status` is about the workload.

Reply:
👍